geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -123,6 +123,7 @@ protected void waitForPrimary(PartitionedRegion.RetryTimeKeeper retryTimer) {
 
   public Set<Integer> clearRegionLocal(RegionEventImpl regionEvent) {
     Set<Integer> clearedBuckets = new HashSet<>();
+    Long clearStartTime = System.nanoTime();
```

Comment:
Changing this from `Long` to `long` will fix the LGTM alert introduced by this PR.